tasks/main.yml

```diff
@@ -37,9 +41,79 @@
     - mssql_upgrade | bool
     - mssql_version | int == 2017
 
+- name: Verify that EL < 8 is not used with mssql_ha_configure=true
+  assert:
+    that:
+      - ansible_distribution in ['CentOS', 'RedHat']
+      - ansible_distribution_version is version('8', '>')
```

Comment:
To include EL 8, should it be `version('8', '>=')`?

Reply:
maybe use `fail` here:
```yaml
- name: Verify that EL < 8 is not used with mssql_ha_configure=true
  fail:
    msg: mssql_ha_configure=true does not support running against EL 7 hosts
  when:
    - ansible_distribution in ['CentOS', 'RedHat']
    - ansible_distribution_version is version('8', '<')
```